kuzzleio/kuzzle-admin-console
# Security-improvement

## What does this PR do ?
Fix #759 by slightly improving security against XSS attacks:
* no more `v-html` in the codebase
* Content-security Policy in the root webpage

## How to test
Try to inject javascript/HTML/CSS code in any part of a document/user/role/profile/notification/environment that is rendered back in the Admin Console and verify it doesn't get evaluated.
